default/scripting/starting_unlocks/items.inf

```diff
@@ -6,11 +6,6 @@ Item type = ShipPart name = "FU_BASIC_TANK"
 Item type = ShipPart name = "CO_OUTPOST_POD"
 Item type = ShipPart name = "CO_COLONY_POD"
 Item type = ShipPart name = "DT_DETECTOR_1"
-Item type = ShipPart name = "FT_BAY_1"
-Item type = ShipPart name = "FT_HANGAR_1"
-Item type = ShipPart name = "FT_HANGAR_2"
-Item type = ShipPart name = "FT_HANGAR_3"
-Item type = ShipPart name = "SR_WEAPON_0_1"
```

Comment:
Self defence, same as Mass Driver, not sure it'll stay there but think it likely and I want to redo the premades so there are some early game Destroyer class ships